time-tracker/src/components/task.jsx

```diff
@@ -0,0 +1,37 @@
+/** @jsxImportSource @emotion/react */
+import { css } from "@emotion/react";
+import { formatTimePassedInSeconds } from "../utils.jsx";
+import { useDispatch } from "react-redux";
+import { changeCurrentTask, pauseAllTasks } from "../redux/reducers.jsx";
+
+export default function Task({
+  name,
+  timePassedInSeconds,
+  taskID,
+  active = false,
```

Comment:
When naming variables that hold boolean values, a good practice is to prefix them with `is`: `isActive`